apps/web/src/app/settings/change-kdf/change-kdf-confirmation.component.html

```diff
@@ -43,7 +43,7 @@
     <button bitButton buttonType="primary" type="submit" [loading]="loading" form="form">
       <span>{{ "changeKdf" | i18n }}</span>
     </button>
-    <button bitButton buttonType="secondary" type="button" data-dismiss="modal">
+    <button bitButton buttonType="secondary" type="button" data-dismiss="modal" bitDialogClose>
```

Comment:
`data-dismiss` is no longer required:

```suggestion
    <button bitButton buttonType="secondary" type="button" bitDialogClose>
```